Sources/TensorFlow/Layers/Recurrent.swift

```diff
@@ -200,6 +200,75 @@ public struct LSTMCell<Scalar: TensorFlowFloatingPoint>: RNNCell {
     }
 }
 
+/// An GRU cell.
+public struct GRUCell<Scalar: TensorFlowFloatingPoint>: RNNCell {
+    public var updateWeight, updateWeight2, resetWeight, resetWeight2, outputWeight, outputWeight2: Tensor<Scalar>
+    public var updateBias, outputBias, resetBias: Tensor<Scalar>
+
+    @noDerivative public var stateShape: TensorShape {
+        TensorShape([1, updateWeight.shape[0]])
```

Comment:
```suggestion
        [1, updateWeight.shape[0]]
```
Use literal initialization when a contextual type exists.